lighthouse-cli/test/smokehouse/test-definitions/mixed-content/expectations.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
(moved into a better named folder)

Reply:
I [moved this in here](https://github.com/GoogleChrome/lighthouse/pull/12819#pullrequestreview-712277151) because in theory there will be other `Issues` smoke tests and this seemed like a good place to put them (the smoke test is `issues-mixed-content` to be able to group with other `issues-*` tests).

I don't feel strongly if it seems better here, but also need to remove the old one :) (and should maybe update the smoke id)

Reply:
OK, makes sense, reverted.